pkgpanda/util.py

```diff
@@ -150,6 +151,38 @@ def get_requests_retry_session(max_retries=4, backoff_factor=1, status_forcelist
     return session
 
 
+def stream_remote_file_with_retries(out_filename, url, retries=4):
```

Comment:
Can we use a known-good retrying library. We already have one on clusters (see [here](https://github.com/dcos/dcos/blob/86a6934657c3d1c22f77d3cd21e112f940988043/ssh/test_integration.py#L13) for an example use).